src/main/java/frc/robot/subsystems/arm/commands/SetArmsPosition.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.subsystems.arm.commands;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.geometry.Translation2d;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
+import frc.robot.subsystems.arm.ArmConstants;
+import frc.robot.subsystems.arm.ArmKinematics;
+import frc.robot.subsystems.arm.ArmSystemModel;
+import frc.robot.subsystems.arm.PrototypeArm;
+
+public class SetArmsPosition extends CommandBase {
+    private final PrototypeArm prototypeArm;
+    private final Translation2d position;
+
+    public SetArmsPosition(PrototypeArm prototypeArm, Translation2d position) {
```

Comment:
Is it necessary to have seperate command for each part of the arm?

Reply:
I think it's mainly for testing, Gaya asked me to add them